doc/rtd/reference/yaml_examples/index_boot.rst

```diff
@@ -0,0 +1,36 @@
+System initialization and boot
+==============================
+
+* :ref:`cce-boot-cmds` using ``bootcmd`` and ``runcmd``
```

Comment:
I think it would make sense, but since I'll be touching all the landing pages again next cycle, I think I'll wait until then to implement this as part of that work. At the very least, I want to include keywords on these index pages to make searching easier (updated my Jira card so I don't forget).